flickit-advice-engine/src/main/java/org/flickit/assessment/advice/application/domain/Target.java

```diff
@@ -0,0 +1,25 @@
+package org.flickit.assessment.advice.application.domain;
+
+import lombok.Getter;
+
+@Getter
+public final class Target {
+
+    private final double currentGain;
+    private final double minGain;
+    private final double neededGain;
```

Comment:
remove this field and add a method named `getRemainingScore` to get their difference.